plugins/modules/github_org_repository.py

```diff
@@ -169,35 +168,39 @@
           manage issues and pull requests without write access.
         type: str
         choices: [pull, push, admin, maintain, triage]
-        default: pull
+        default: push
   collaborators:
     description: |
       Repository collaborators with their permissions
     type: list
     elements: dict
     options:
-      slug:
-        description: Team slug
+      username:
```

Comment:
weird

Reply:
doc was a little bit wrong

Reply:
weird that sanity didn't complain. Is it running properly at all?